homeassistant/components/homekit/type_media_players.py

```diff
@@ -257,16 +257,17 @@ def __init__(self, *args):
         state = self.hass.states.get(self.entity_id)
 
         self.support_select_source = False
+        self._supports_play_pause = False
 
         self.sources = []
 
         # Add additional characteristics if volume or input selection supported
```

Comment:
```suggestion
```

It doesn't make sense anymore so let's just delete the comment as keeping a list of characteristics isn't unique to  this accessory type.